src/components/AvatarPopUp/AvatarPopUp.component.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { Avatar, Box, Popover, Typography } from '@mui/material';
+
+import { useStyles } from './AvatarPopUp.styles';
+import { AvatarPopUpProps } from './AvatarPopUp.types';
+
+export const AvatarPopUp = (props: AvatarPopUpProps) => {
+    // Extract the generated CSS classes using the custom TSS (Tailwind-Styled System or similar) hook
```

Comment:
```suggestion
    // Styles
```

comments should be short and easy to understand, explain in case some logic is there